ironfish-cli/src/commands/airdrop/post.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Flags } from '@oclif/core'
+import fs from 'fs/promises'
+import { IronfishCommand } from '../../command'
+
+export default class AirdropPostTransactions extends IronfishCommand {
+  static description = `Drop coins to testnet participants`
```

Comment:
Maybe a slight description change, doesn't really matter for these commands though!

```suggestion
  static description = `Post transactions for testnet participants`
```